cmd/root.go

```diff
@@ -20,6 +21,15 @@ const (
 	DefaultDateFormat = "2006-01-02"
 )
 
+var (
+	// defaultStart is the default start date to query pull requests if none is
+	// specified.
+	defaultStart string
+	// defaultEnd is the default end date to query pull requests if none is
+	// specified.
+	defaultEnd string
```

Comment:
☝️ These global variables expose the default values to tests, such that `gock` can be used to mock the resulting GraphQL queries.